src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PurposesApiServiceImpl.scala

```diff
@@ -482,6 +482,8 @@ final case class PurposesApiServiceImpl(
 
     val result: Future[PurposeVersionResource] = for {
       purposeUUID    <- purposeId.toFutureUUID
+      _              <- catalogProcessService.getEServiceById(purposeUpdateContent.eserviceId)
+      _              <- tenantProcessService.getTenant(purposeUpdateContent.consumerId)
```

Comment:
[Same](https://github.com/pagopa/interop-be-purpose-process/pull/169/files#r1266751056)

Reply:
These are already checked at the process level.
We can remove them.